app/views/partners/_form.html.erb

```diff
@@ -22,16 +22,16 @@
                   <%= f.input_field :partner_group_id, collection: @partner_groups, class: "form-control", include_blank: "None" %>
                 <% end %>
               </div>
-
+                  #changed
               <div data-partner-reminder-form>
-                <%= f.input :send_reminders, label: "Do you want to send deadline reminders to them every month?", wrapper: :input_group do %>
+                <%= f.input :send_reminders, label: "Do you want this partner to receive emails for distributions and reminders from the system?", wrapper: :input_group do %>
                   <%= f.check_box :send_reminders, {class: "input-group-text", id: "send_reminders", data: { 'partner-reminder-form-checkbox': '' } }, "true", "false" %>
-                <% end %>
-
+                 <% end %> 
+                  
                 <div class='mb-2' data-partner-reminder-form-nested-form>
                   <div class="flex items-center bg-yellow-300 text-sm font-bold px-4 py-3" role="alert">
-                    <p>Reminders will be sent out according to the settings of their partner group OR organization. This won't work unless you've set up at least one.</p>
-                  </div>
+                    <p>Please note that reminders will be sent out according to the settings of their partner group OR organization. This won't work unless you've set up at least one.</p>
+                  </div> #changed
```

Comment:
Same as above. This is going to result in extra markup on the page.

Reply:
I have removed the comments and pushed the code...Please check it once.

Reply:
Awesome! Looks great!